packages/components/src/templates/table.stories.ts

```diff
@@ -0,0 +1,596 @@
+import '../solid-components';
+import { html } from 'lit-html';
+import { ifDefined } from 'lit/directives/if-defined.js';
+
+/**
+ * The `sd-table-cell` component offers basic styling for table cells.
+ * It is designed to be used in conjunction with the `sd-table` component.
+ * 
+
+ * These examples demonstrate the usage of sd-table-cell in various contexts.
+ * The examples are intended solely for illustrating how sd-table-cell can be used to style tables.
+ * The data generation and table sorting logic should not be used in production environments.
+ *
+ */
+
+export default {
```

Comment:
Hide default, pls since it is shown in the intro

Reply:
Updated.